astropy/utils/console.py

```diff
@@ -658,9 +658,9 @@ def _update_ipython_widget(self, value=None):
             self._widget.value = 0
 
         # Calculate percent completion, and update progress bar
-        percent = (value/self._total) * 100
-        self._widget.value = percent
-        self._widget.description =' ({0:>6s}%)'.format('{0:.2f}'.format(percent))
+        percent = (value/self._total)
```

Comment:
I would either keep the multiplication here, as this is not the percent but the fractional value in this form or rename the variable to ``frac`` as above.